### PR TITLE
Geostyler update changes

### PR DESCRIPTION
**Pull request by FilipLeitner**
## Description

Update styler component based on new Geostyler changes
-  Use fillOpacity parameter instead of opacity for fill of mark symbolized
-  Remove 'Not visualized correctly' warning from font weight, style of text symbolized and graphicFill of fill Symbolizer
- Update styler UI to indicate that style changes are being saved

## Related issues or pull requests

#3677 

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Dependency updates
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe)
- [ ] I am unsure (we'll look into it together)

## Do you introduce a breaking change?

- [ ] Yes
- [ ] No
- [ ] I am unsure (no worries, we'll find out)

## Checklist

- [x] I understand and agree that the changes in this PR will be licensed under the [MIT License]
- [ ] I have followed the [guidelines for contributing](https://github.com/hslayers/hslayers-ng/blob/master/CONTRIBUTING.md)
- [ ] The proposed change fits to the content of the [code of conduct](https://github.com/hslayers/hslayers-ng/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have added or updated tests and documentation, and the test suite passes (run `npm test` locally)
- [ ] I'm lost; why do I have to check so many boxes? Please help!

### CODE REVIEW

File: projects/hslayers/src/components/save-map/layman-utils.ts [hunk 1]
```diff
@@ -86,3 +90,14 @@ export function isAtLeastVersions(ep: HsEndpoint, version: string) {
 
   return true;
 }
+
+/**
+ * Wait until layer synchronization is complete
+ * @param layer - OL Layer
+ */
+export async function awaitLayerSync(layer: Layer): Promise<any> {
```
Comment (jmacura) [inline projects/hslayers/src/components/save-map/layman-utils.ts:98]:
```suggestion
export async function awaitLayerSync(layer: Layer): Promise<boolean> {
```

Not sure which type restriction would work best here. Might be `<true | undefined>`? Also removing the return type and let the TS compiler to infer the type is OK here I guess.

**Review by jmacura — CHANGES_REQUESTED** [review]
Minor type change requested. Thanks!

File: projects/hslayers/src/components/save-map/layman-utils.ts (hunk 1 above)
Comment (FilipLeitner) [reply projects/hslayers/src/components/save-map/layman-utils.ts:98]: Boolean is fine I think

**Comment by jmacura** [comment]
Can you resolve the conflicts pls, @FilipLeitner ?

**Review by jmacura — APPROVED** [review]

**Pull request merged**